docs/hypernode-platform/autoscaling/how-does-horizontal-autoscaling-work.md

```diff
@@ -57,6 +57,8 @@ Horizontal autoscaling is available on all the Falcon cloud plans (OpenStack).
 
 Next to the provider, horizontal autoscaling does have a couple of additional requirements.
 
+- **Note:** Staging ports (8888, 8443) will not be available when horizontal autoscaling is enabled.
```

Comment:
````suggestion
```{note}
Staging ports (8888, 8443) will not be available when horizontal autoscaling is enabled.
```
````